index.js

```diff
@@ -73,6 +77,144 @@ This is a ` + type + ` of #` + pr.pull_number + `.
 
 module.exports = (app) => {
 
+  app.on(["pull_request.opened", "pull_request.synchronize"], async (context) => {
+    const pr = context.pullRequest();
+
+    if (pr.owner != 'vitessio' || pr.repo != 'vitess') {
+      return
+    }
+
+    const per_page = 100;
+    let cont = true;
+    let files = []
+    for (let page = 1; cont; page++) {
+      let currentFiles = await context.octokit.rest.pulls.listFiles({
+        owner: pr.owner,
+        repo: pr.repo,
+        pull_number: pr.pull_number,
+        per_page: per_page,
+        page: page,
+      });
+      if (currentFiles.data.length < per_page) {
+        cont = false;
+      }
+      files = files.concat(currentFiles.data)
```

Comment:
any reason we first concatenate all and then look at the files? why not just look at the files in this loop, so we don't have to fetch anything else

Reply:
That's a good point, I'll open an issue for that and optimize it later